composer.json

```diff
@@ -86,7 +86,7 @@
         "doctrine/doctrine-migrations-bundle": "^1.3.0",
         "doctrine/migrations": "^1.6.0",
         "doctrine/persistence": "~1.2.0",
-        "elasticsearch/elasticsearch": "^6.0.1",
+        "elasticsearch/elasticsearch": "^7.5.0",
```

Comment:
Just a question: version 7.6.1 (https://packagist.org/packages/elasticsearch/elasticsearch#v7.6.1) doesn't work?

Reply:
It is same "problem" like version of Elasticsearch and Kibana above. But Major and Minor version have to be same for Kibana, Elasticsearch and this package....